### PR TITLE
Update img.md

### PR DESCRIPTION
**Pull request by ethan-master-coding**
### Description

Added that you use the `img` tag for gifs

### Type of Change

- Editing an existing entry (fixing a typo, bug, issues, etc)
- Updating the documentation

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/html/concepts/tags/terms/img/img.md
Comment (Dusch4593) [inline content/html/concepts/tags/terms/img/img.md:14]: This seems like sort of an awkward place to put this? The `` tag supports a multitude of formats, including `.gif`. Honestly, this entire entry could use a polish. Here's what I suggest for each section.

This line:

```suggestion
The `<img>` tag displays an image on the web page.
```

The sentence "It can also be used inside an anchor tag to create an image link." should be incorporated into the "Example" section below. We can also link "anchor tag" to the relevant entry (https://www.codecademy.com/resources/docs/html/tags/a).

**Syntax**:
All the list-items should be turned into complete sentences with periods (`.`) at the end.
  * The one about the `src` attribute should include a short list of common supported formats:
```
- `src`: the URL or filename of the image that supports a litany of formats (GIF, JPEG, PNG, etc.).`
```

**Example**:
Can we include an image of what the rendered `logo.png` actually looks like? This can be done by (1) adding the image to the `media/` folder and (2) referencing here via markdown with "alt-text-here" (no quotes, of course).

**Comment by ethan-master-coding** [comment]
@Dusch4593 Accepted the suggestion and made that change to the bullet. I will make the example later today or tomorrow.

**Comment by ethan-master-coding** [comment]
@Dusch4593 For when I do this its just an image on the page?

**Comment by Dusch4593** [comment]
@ethan-master-coding I see you uploaded the image! 👍 

A few things, though:

* Let's keep the filename to lowercase and rename it to either `cc_logo.png` or `logo.png`.
* Let's also make sure it matches with the `src="..."` attribute.
* Under the example code snippet (below line # 42), let's reference it like so: \!\[Screenshot of rendered image](https://raw.githubusercontent.com/Codecademy/docs/main/media/cc_logo.png)

**Comment by ethan-master-coding** [comment]
@Dusch4593 Ok. I was going to make a second example just displaying the image.

File: content/html/concepts/tags/terms/img/img.md
Comment (Dusch4593) [inline content/html/concepts/tags/terms/img/img.md:45]:
```suggestion
![Codecademy logo](https://raw.githubusercontent.com/Codecademy/docs/main/media/cc_logo.png)
```

Comment (Dusch4593) [reply content/html/concepts/tags/terms/img/img.md:45]: Relative paths to files in the `media/` folder will ultimately not render in the live entries.

File: content/html/concepts/tags/terms/img/img.md
Comment (Dusch4593) [inline content/html/concepts/tags/terms/img/img.md:27]:
```suggestion
- `height`: the height of the image, measured in pixels.
- `width`: the width of the image, measured in pixels.
```

**Comment by Dusch4593** [comment]
In the meta-`Description` on line # 3, let's replace the current text with "Displays an image on the web page."

**Comment by ethan-master-coding** [comment]
@Dusch4593 ll edits look good. I accepted the changes and made the change you accepted.

**Comment by Dusch4593** [comment]
@ethan-master-coding Appreciate it! 🙏 

I'm gonna ask @yangc95 to take a look at the edits made thus far and we can merge afterwards.

**Comment by ethan-master-coding** [comment]
@Dusch4593 Not related to the PR but once my open PRs are merged it will tie me for 10th place in # of contributions to the docs.

File: content/html/concepts/tags/terms/img/img.md
Comment (yangc95) [reply content/html/concepts/tags/terms/img/img.md:24]: I linked to the image entry here~ I also noticed that **.gif** files are not mentioned in the [Images entry](https://www.codecademy.com/resources/docs/html/images). Great catch and addition btw from the forums. Loving the help you're giving to our community ☺️

File: content/html/concepts/tags/terms/img/img.md
Comment (yangc95) [inline content/html/concepts/tags/terms/img/img.md:29]: Are you planning on adding more examples? I'm not opposed to that, but if we're not adding another "Example 2" below, let's change this back to "Example" 👍

**Review by yangc95 — COMMENTED** [review]
📥 Review 2️⃣ ✅

**Comment by ethan-master-coding** [comment]
@yangc95 I'm about to go to bed. I will do these tomorrow morning.

**Comment by ethan-master-coding** [comment]
@yangc95  All edits look good. I accepted the changes. To bad other SUs or miss don't contribute as often as I do.

**Review by Dusch4593 — APPROVED** [review]
@ethan-master-coding Things looks good from my end but wanted to mention that we should always have _**at least two meta-`Tags`**_ near the top.

File: content/html/concepts/tags/terms/img/img.md
Comment (yangc95) [inline content/html/concepts/tags/terms/img/img.md:23]:
```suggestion
The HTML image tag, `<img>`, has no closing tag. An image element cannot have other tags inside of it. It uses attributes to set properties, some common attributes are:
```

After reading over this again, one last thing ☝️

Comment (yangc95) [reply content/html/concepts/tags/terms/img/img.md:23]: @ethan-master-coding Also, go ahead and update the branch! Going to merge this after

**Comment by ethan-master-coding** [comment]
@yangc95 ready to merge.

**Comment by yangc95** [comment]
@ethan-master-coding @Dusch4593 Could you both run `git config --global user.email your.email` in your terminal?

**Comment by ethan-master-coding** [comment]
@yangc95 I cant right now. Plus I never use the terminal so I'm not sure if that's the problem (at least for me)

**Comment by ethan-master-coding** [comment]
@yangc95 me taking away and adding a single character back to force the checks to run again worked. They all passed now.

**Comment by Dusch4593** [comment]
> @ethan-master-coding @Dusch4593 Could you both run `git config --global user.email your.email` in your terminal?

Afk right now but will do so ASAP.

**Review by yangc95 — APPROVED** [review]

**Comment by yangc95** [comment]
@Dusch4593 you're good! Thanks, guys 😌  @ethan-master-coding 

Changes may require some time, but should appear here: https://www.codecademy.com/resources/docs/html/tags/img

**Pull request merged**
